sonar-db/src/main/java/org/sonar/db/user/UserDao.java

```diff
@@ -102,6 +102,10 @@ public UserDto selectActiveUserByLogin(DbSession session, String login) {
     return executeLargeInputs(logins, session.getMapper(UserMapper.class)::selectByLogins);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this use of "selectByLogins"; it is deprecated. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)